core/src/Slay/Prim.hs

```diff
@@ -82,6 +86,18 @@ data PrimCurve g =
       curveDirection :: g Direction
     }
 
+
+-- | circle adt
+data PrimCircle g = Circle
+    { circleColor :: g (Maybe Color)
+    , circleCenterX :: Double
+    , circleCenterY :: Double
+    , circleRadius :: Double
```

Comment:
I called them like this since they are X an Y positions of the center of the arc, if you got better names I will be glad to change it